lib/Frontend/Frontend.cpp

```diff
@@ -315,6 +316,18 @@ bool CompilerInstance::setUpModuleLoaders() {
                                                        getDependencyTracker());
     Context->addModuleLoader(std::move(PIML));
   }
+
+  if (Invocation.getLangOptions().DebuggerSupport) {
```

Comment:
I think you should have a separate option for this for now. We don't want to mess up normal debugging while this is coming online.

Reply:
I'll double-check this, but my understanding is that it is under LLDB's control to create any ClangImporters, so I'll hide the option under an LLDB user setting instead. Meanwhile, when you're invoking swift with --debugger-support you're explicitly asking for trouble, I mean, "testing the compiler".

Reply:
I ended up creating a separate option, both for swift, and a setting for LLDB.